js/views/pauseScreen.js

```diff
@@ -104,86 +107,60 @@ class PauseScreen extends React.Component {
     this.props.handleVrPlayerClick();
   }
 
-  /**
-   * call handleTouchEnd when touchend was called on selectedScreen and videoType is Vr
-   * @param {Event} e - event object
-   */
-  handleTouchEnd(e) {
-    if (this.props.controller.state.controlBarVisible) {
-      var shouldToggle = false;
-      if (this.props.controller.videoVr) {
-        e.preventDefault();
-        if (!this.props.isVrMouseMove) {
-          shouldToggle = true;
-        }
-      } else {
-        shouldToggle = true;
-      }
-      if (shouldToggle) {
-        this.props.controller.togglePlayPause(e);
-      }
-    }
-  }
-
-  /**
-   * call handleVrTouchEnd when touchend was called on selectedScreen and videoType is Vr
-   * @param {Event} e - event object
-   */
-  handleVrTouchEnd(e) {
-    this.props.handleVrPlayerMouseUp(e);
-  }
-
   /**
    * remove the button on pause screen for correct checking mouse movement
    */
   hideVrPauseButton() {
-    if (this.props.controller.videoVr) {
-      var pauseButton = document.getElementById('oo-pause-button');
-      setTimeout(function() {
-        if (pauseButton) {
-          pauseButton.style.display = 'none';
-        }
-      }, 1000);
+    if (!(this.props.controller.videoVr && this.pauseButton)) {
+      return;
     }
+    setTimeout(() => this.pauseButton.style.display = 'none', 1000);
```

Comment:
Is there a reason why we modify the element's style directly instead of using `setState()`?

Reply:
it is possible to use setState, but the idea was to use the same time as in css, if we use setState we will have a delay